osu.Game/Graphics/UserInterfaceV2/OsuPopover.cs

```diff
@@ -51,5 +54,24 @@ protected override void PopOut()
             this.ScaleTo(0.7f, scale_duration, Easing.OutQuint);
             this.FadeOut(fade_duration, Easing.OutQuint);
         }
+
+        public bool OnPressed(GlobalAction action)
+        {
+            if (State.Value == Visibility.Hidden)
+                return false;
+
+            if (action == GlobalAction.Back)
+            {
+                Hide();
+                return true;
+            }
+
+            return false;
+        }
+
+        public void OnReleased(GlobalAction action)
+        {
+            throw new System.NotImplementedException();
```

Comment:
This probably shouldn't be throwing, should it? Usual operating practice was just leaving as a no-op, I'm pretty sure...

Quite weird that this doesn't actually seem to trigger on the esc press, really.

Reply:
Was an oversight, should have been removed.